.evergreen/config.yml

```diff
@@ -81,6 +81,10 @@ functions:
               export SSL=${SSL}
               export TOPOLOGY=${TOPOLOGY}
               export MONGODB_VERSION=${MONGODB_VERSION}
+
+              if [ "Windows_NT" != "$OS" ]; then
```

Comment:
Windows machines complain if we modify this, so I restricted it to macOS and Linux. macOS machines are the main culprit for our too many files errors anyways.